yt_idefix/data_structures.py

```diff
@@ -344,7 +344,7 @@ def _parse_parameter_file(self):
             [fdata[f"xr{idir}"][-1] for idir in "123"], dtype="float64"
         )
 
-        self.current_time = fdata["time"]
+        self.current_time = self.quan(fdata["time"], "code_time")
```

Comment:
Just to be sure: this is in the `IdefixDmpDataset` class, which isn't a parent to `PlutoVtkDataset`, so it's not directly related to the bug you were addressing, but it's a similar issue that you're fixing while you're at it, correct ?

Reply:
Oh I did it because you said you want current_time to be unyt_quantity intially, so I thought you forgot to make it here. Since idefix doesn't include unit definitions, the bug I'm addressing is not a bug indeed for idefix dataset.

Reply:
it's fine, just wanted to make sure this wasn't intended for `IdefixVtkDataset`. Che che !

Reply:
Btw, I'm curious about what does `che che` mean? Is that a French phrase or english? 😆 Just cannot find its meanning.

Reply:
Ha ! That was a lazy attempt at using the latin alphabet for "谢谢" ;)